eras/shelley-ma/impl/cardano-ledger-shelley-ma.cabal

```diff
@@ -2,10 +2,9 @@ cabal-version: 3.0
 
 name:                cardano-ledger-shelley-ma
 version:             0.1.0.0
-synopsis:            Shelley ledger with multiasset and time lock support.
+synopsis:            Deprecated package that used to implement Allegra and Mary eras.
```

Comment:
Just curious if there is a particular reason to leave the deprecated ones around, or just common sense?
Any idea of when to remove them?

Reply:
This is to help downstream packages migrate to new packages. We on=always try to give users some time to migrate and then we can get rid of deprecated stuff